File: semestre-1/algoritmos/lista-1/python/MR-Alg-01-Ex-12.py
import math

r = float(input("Digite o raio (em cm): "))

area = math.pi * r**2
volume = 4/3 * math.pi * r**3

print(f"Área de um círculo com raio {r} = {area}cm²")
print(f"Volume de uma esfera com raio {r} = {volume}cm³")

File: semestre-1/algoritmos/lista-1/c/MR-Alg-01-Ex-12.c
#include <stdio.h>
#include <math.h>

#define PI 3.14159265358979323846

void main() {
    double r;

    printf("Digite o raio (em cm): ");
    scanf("%lf", &r);

    double area = PI * pow(r, 2);
    double volume = 4.0 / 3.0 * PI * pow(r, 3);

    printf("\nÁrea de um círculo com raio %lf = %.2lf cm²\n", r, area);
    printf("Volume de uma esfera com raio %lf = %.2lf cm³\n", r, volume);
}